integration-tests/USECASES.MD

```diff
@@ -49,6 +49,8 @@ Basic Use Cases described above are verified in all the domain configurations. A
 | Domain with default sample values | create domain using mostly default values for inputs |	
 | Domain home in image using WLST | cluster scaling |
 | Domain home in image using WDT  | Configured cluster and cluster scaling |
+| Domain with 2 configured cluster created using WLST | Multi Configured cluster |
```

Comment:
add under "Full test Configuration & Use Cases - Runs Quick test Configuration & Use cases and the below" in Domain Configuration and remove from here. 
```
Domain on PV with two configured clusters
Domain on PV with one configured cluster and one dynamic cluster
```

Reply:
Done